### ISSUE TITLE
[Feature Request] - Adguard Mobile Annoyances list

### ISSUE THREAD

**Issue by techgamers2** [r-a-y/mobile-hosts#29]
can we get adguards mobile app banner list coz app banners r not been block when using adguard mobile ads list 

Aguard : [Mobile Ads Banner](https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_20_Annoyances_MobileApp/filter.txt)

**Comment by r-a-y** [comment]
> edit : tried to generate list using converter.php got this result

Yeah the converter has limitations. I explain that in this comment: https://github.com/r-a-y/mobile-hosts/issues/3#issuecomment-512852976

If you're using personalDNSfilter, perhaps you can track the domains that the banners are using and add them to your blocklist.

As for the three domains, `cdn.branch.io` is covered in AdguardTracking.txt. The other two are not included in the lists I cover.
